core/build.gradle.kts

```diff
@@ -32,13 +32,19 @@ java {
 }
 
 dependencies {
-    compileOnlyApi("com.github.Minestom:Minestom:7be96b7679")
-    compileOnly("space.vectrix.flare:flare-fastutil:2.0.1")
-    compileOnly("com.google.code.gson:gson:2.9.0")
+    compileOnlyApi("com.github.Minestom:Minestom:${rootProject.extra["minestomVersion"]}")
```

Comment:
Version catalogue should be used instead of variables.

Reply:
Could you give a example?